### ISSUE TITLE
[BUG] PyPI workflow in actions does not work

### ISSUE THREAD

**Issue by hmasdev** [hmasdev/pyjpboatrace#35]
## Describe the bug
PyPI workflow in actions, which runs when released, returns exit code 1 during uploading .whl and .tar.gz with twine.

https://github.com/hmasdev/pyjpboatrace/actions/runs/1103843756

## Environment
 - OS: Ubuntu 20.04.2 LTS
 - Python Version: 3.7.11 (actions/setup-python@v1)

## To Reproduce (Probabily)
1. Go to https://github.com/hmasdev/pyjpboatrace/releases
2. Click on "Draft a new release"
3. Input tag version, title and description.
4. Click on "Publish release", which triggers PyPI workflow.
5. Error during PyPI workflow in Upload step as follows:
   ```bash
   Traceback (most recent call last):
      File "/home/runner/.local/share/virtualenvs/pyjpboatrace-cg8vEFYy/bin/twine", line 5, in <module>
        from twine.__main__ import main
      File "/home/runner/.local/share/virtualenvs/pyjpboatrace-cg8vEFYy/lib/python3.8/site-packages/twine/__init__.py", line 27, in <module>
        import importlib_metadata
   ModuleNotFoundError: No module named 'importlib_metadata'
   Error: Process completed with exit code 1.
   ```

## Expected behavior

Twine succeeds in uploading .whl and .tar.gz to pypi server.

```bash
Uploading distributions to https://upload.pypi.org/legacy/
Uploading pyjpboatrace-0.2.0-py3-none-any.whl
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 63.5k/63.5k [00:02<00:00, 24.2kB/s]
Uploading pyjpboatrace-0.2.0.tar.gz
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50.3k/50.3k [00:01<00:00, 31.2kB/s] 

View at:
https://pypi.org/project/pyjpboatrace/0.2.0/
```

## Problems
If this bug is not fixed, we can not upload this package to pypi automatically, that is, manual operation is required  when uploading.

## Reasons (Optional)
Checking the log, I found pipenv uses python 3.8.10 as follows:

```bash
Creating a virtualenv for this project...
Pipfile: /home/runner/work/pyjpboatrace/pyjpboatrace/Pipfile
Using /usr/bin/python3.8 (3.8.10) to create virtualenv...
created virtual environment CPython3.8.10.final.0-64 in 694ms
```

This is probably a reason for this bug. 

See https://stackoverflow.com/a/59734959/16567832

**Comment by hmasdev** [comment]
The reason why Python 3.8 is used when creating a virtual environment even though Python 3.7 is specified in https://github.com/hmasdev/pyjpboatrace/blob/main/.github/workflows/pypi.yaml is probably because the default python version is 3.8 and any version is not specified in Pipfile or Pipfile.lock.

**Comment by hmasdev** [comment]
Propose to fix the python version to 3.9 for development.

**Comment by hmasdev** [comment]
[The document of pipenv](https://pipenv.pypa.io/en/latest/basics/#general-recommendations-version-control) recommends not to keep Pipfile.lock in version control if multiple versions of Python are being targeted.

This project is a package for python 3.7, 3.8 and 3.9. So the version of Pipfile.lock should not to be maintained in this project.
